tests/cases/fourslash/formattingOnSingleLineBlocks.ts

```diff
@@ -1,16 +1,11 @@
 /// <reference path='fourslash.ts' />
 
-/////*1*/class C
-////{}/*2*/
-/////*3*/if (true)
-////{}/*4*/
+////class C

```

Comment:
Many constructs are already tested:
* object literals: `formattingOnObjectLiteral.ts` (see marker 26 for an empty object literal)
* type literals: `formatAnyTypeLiteral.ts`
* lambda: `formattingOnMultiLineBlockConstructs.ts` (marker 10 has an empty lambda).

We could reorganize these tests to be a bit more systematic, but it's unclear this would be worth the effort right now.